src/main.test.js

```diff
@@ -79,76 +79,3 @@ describe("LogflareHttpClient", () => {
         )
     })
 })
-
-describe("LogflareHttpClient with options", () => {
-    let httpClient
-    let nativeFetch
-
-    beforeAll(() => {
-        nativeFetch = global.fetch
-    })
-
-    beforeEach(() => {
-        httpClient = new LogflareHttpClient({
-            apiKey: testApiKey,
-            sourceToken: testSourceToken,
-            apiBaseUrl: testBaseUrl,
-            transforms: {jsNumbers: true},
-        })
-    })
-
-    afterEach(() => {
-        global.fetch.mockClear()
-    })
-
-    afterAll(() => {
-        global.fetch = nativeFetch
-    })
-
-    it.skip("transforms js numbers if configured", async () => {
```

Comment:
It seemed like this skipped test is related to typecasting, but I'm not entirely sure. Please let me know if I should undo this change!